applications/liberland.md

```diff
@@ -0,0 +1,345 @@
+
+The hitherto unseen benefit is that voting on-chain can stay permanent. An online system's relatively low transaction costs allow a polity to use higher thresholds for adopting a motion. "Paper" referenda as implemented in current States usually rely on artificially low quora, breaking the original assumption of a "majority rule". We hope to be able to unleash the true power of the people by giving them the flexibility and the security of a public blockchain. 
+
+
+#### Executive Power    
+As required by the standards of modern Statehood, the Executive (vested in a central organ, the Cabinet) is voted in and removed by Congress. The standard of a "fully responsible" government requires States to adopt such a system or face criticism as "not fully democratic".    
+Our Executive is a fork of modern parachain councils such as Polkadot's Council, though heavily modified. It is composed of four ministers led by one prime minister. Other adoptees can expand the system, designed to handle complex administrative processes such as regulations, making decisions and even dispute resolution.
+
+
+Connection to extrinsic systems, standard or customized per-state via API will enable the system to integrate seamlessly into an existing State information system. We build e-government for Liberland, partly inspired by Estonia but fully Web-3 based.
+The powers of the Executive in Liberland are to govern directly on-chain within the limits of Congress's laws and off-chain as well in the metaverse and later the real-life state.
+    
+    
+More complex State actors might create specialized parachains which would govern their municipalities or even byzantine central bureaucracies. Given that States generate traffic as a matter of course, such systems are very interesting for Validators and Nominators and virtually non-existent in the current space.
+ 
+
+
+#### Treasury   
+The Treasury for LLM, unlike that of Polkadot and like treasuries of extant States, operates without the burn mechanism. LLD follows the same rules as Polkadot, and so there is a burn mechanism to offset the “infinite” (really: algorithmic) supply. 
+As the LLM tokenomics suppose a deflationary token, holding LLM in the treasury will help finance running the State. The System distributes a significant portion of the treasury among the Members of Congress. Through this, the tokens get directly into the hands of the Voters (Citizens). Using Pallet-llm the treasury is the owner and manager of several on-chain assets, allowing us to expand the base functionality to have the treasury securely manage on-chain assets for us. Decentralizing trust in a true way.
+
+
+#### Company Registration Pallet    
+The company registration pallet works similar to government company registrar authorities. This pallet will handle the registration of companies in a truly decentralized way. Allowing anyone to apply for registering on-chain companies
```

Comment:
The Registrar of Companies will be an Agency of the Government. It will evaluate the contents of the Application. Later, we might make this decentralized (i.e. Validators will judge it based on a game theory principle, can message you more details on Matrix), but at first, we opt for a centralized solution, as we favour incremental approach and want to first build and deploy the basis.

Reply:
updated it